src/monorepo-shipit/config/__tests__/example-relay.test.js

```diff
@@ -19,7 +19,6 @@ testExportedPaths(path.join(__dirname, '..', 'example-relay.js'), [
 
   // invalid cases:
   ['src/example-relay/.babelrc.js', undefined], // correctly deleted
-  ['src/example-relay/jest.config.js', undefined], // correctly deleted
```

Comment:
@jaroslav-kubicek Guess who wrote this test ensuring `jest.config.js` is never going to be published via Shipit? 😅

Reply:
nooo :)